recipes/traja/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "traja" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 873c671ccb48a79b1e6e48c70ab9b43819f96241bb339aabacc1ed2e6afd3896
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python>=3
+  run:
+    - python>=3
```

Comment:
We want a space between python and `>=3`

Reply:
Space added with e22fe13. Ready for merging.